x-pack/plugins/security_solution/public/hosts/components/hosts_table/index.tsx

```diff
@@ -129,9 +134,25 @@ const HostsTableComponent: React.FC<HostsTableProps> = ({
   );
   const riskyHostsFeatureEnabled = useIsExperimentalFeatureEnabled('riskyHostsEnabled');
 
+  const dispatchSeverityUpdate = useCallback(
+    (s: HostRiskSeverity) => {
+      dispatch(
+        hostsActions.updateHostRiskScoreSeverityFilter({
+          severitySelection: [s],
+          hostsType: type,
+        })
+      );
+      navigateToApp(APP_UI_ID, {
```

Comment:
FWIW: there's a hook called `useNavigateTo` that makes navigation a little bit more trivial.